hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -125,16 +122,16 @@ public StorageBasedLockProvider(final LockConfiguration lockConfiguration, final
             UUID.randomUUID().toString(),
             lockConfiguration.getConfig(),
             LockProviderHeartbeatManager::new,
-            getLockService(),
+            getStorageLockService(),
```

Comment:
rename to getStorageLockClient(). 
again, do remember to always align entities w/ getters.